>From dfcf6d3bc4c98c624cbc0f724afca8bc97117a49 Mon Sep 17 00:00:00 2001
From: Linn Crosetto <linn@hpe.com>
Date: Mon, 12 Sep 2016 15:43:34 -0600
Subject: [PATCH] efi-readvar: add support for MokListRT, MokListXRT

shim creates copies of boot service variables MokList and MokListX in
runtime variables MokListRT and MokListXRT. Add support for these to
efi-readvar and remove MokList since it is not accessible.

Signed-off-by: Linn Crosetto <linn@hpe.com>
---
 efi-readvar.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/efi-readvar.c b/efi-readvar.c
index 816b306..12c644c 100644
--- a/efi-readvar.c
+++ b/efi-readvar.c
@@ -115,9 +115,9 @@ parse_db(const char *name, uint8_t *data, uint32_t len, int sig, int entry)
 int
 main(int argc, char *argv[])
 {
-  char *variables[] = { "PK", "KEK", "db", "dbx" , "MokList" };
+	char *variables[] = { "PK", "KEK", "db", "dbx" , "MokListRT", "MokListXRT" };
 	char *progname = argv[0], *var = NULL, *file = NULL;
-	EFI_GUID *owners[] = { &GV_GUID, &GV_GUID, &SIG_DB, &SIG_DB, &MOK_OWNER };
+	EFI_GUID *owners[] = { &GV_GUID, &GV_GUID, &SIG_DB, &SIG_DB, &MOK_OWNER, &MOK_OWNER };
 	int i, found = 0, sig = -1, entry = -1, fd;
 
 	while (argc > 1 && argv[1][0] == '-') {
-- 
2.8.0.rc3

